### DISCUSSION TITLE
Comet backup can't access webdav mount

### DISCUSSION THREAD

**Discussion by plancked** [iterate-ch/cyberduck#15658] [Q&A]
Hi,

I'm trying to setup comet backup to backup on the drive B:/ which is a mountain duck mount (webdav).
But comet can't access it:

```
Σαβ Φεβ 24 19:19:40 2024	Error accessing storage path '\\*****-MISC\******-nas-webdav-comet\comet\': CreateFile \\****-MISC\*****-nas-webdav-comet\comet\: The filename, directory name, or volume label syntax is incorrect.. Additionally, creating the directory failed because: mkdir \\****-MISC: The specified path is invalid.
```

Any idea how to solve this?
Thanks
Alex

**Comment by tatze96** [comment]
Is comet backup used as a service application?

Mountain Duck drives are limited to the interactive user session, which is why Mountain Duck can't be used as a service application, and other service applications can't access the mounted drives.
